core/src/apps/cardano/sign_tx.py

```diff
@@ -482,27 +483,27 @@ def _cborize_shelley_witnesses(
     shelley_witnesses = []
 
     # include only one witness for each path
-    paths = set()
+    paths: List[List[int]] = list()
```

Comment:
`[]` is preferred for empty list

Reply:
https://github.com/trezor/trezor-firmware/pull/1493/commits/3d6644da227656edc3a5a29cf970e05c9f9d4c25